packages/devtools_app/lib/src/network/flutter/network_screen.dart

```diff
@@ -76,20 +85,23 @@ class NetworkScreenBodyState extends State<NetworkScreenBody> {
     return Row(
       children: [
         recordButton(
+          key: NetworkScreen.recordButtonKey,
           recording: isRecording,
           minIncludeTextWidth: minIncludeTextWidth,
-          onPressed: () => _networkController.startRecording(),
+          onPressed: () => networkController.startRecording(),
```

Comment:
drive by cleanup. you can just write tearoffs here instead. So 
`networkController.startRecording` 
instead of
`() => networkController.startRecording()`

@pq  is there an existing lint that can help with this? "Lint name: prefer_method_tearoffs"

Reply:
of course, in the general case the tearoff could be be different but in this case networkController is final so the two are known to be equivalent barring weird subclasses.

Reply:
Good point, not sure why I did it this way...